libs/ledger-live-common/src/hw/actions/startExchange.ts

```diff
@@ -95,11 +98,38 @@ export const createAction = (
     const [state, setState] = useState(initialState);
     const reduxDeviceFrozen = useFrozenValue(reduxDevice, state.freezeReduxDevice);
 
-    const { exchangeType } = startExchangeRequest;
+    const { exchangeType, provider, exchange } = startExchangeRequest;
+    const requireLatestFirmware = true;
```

Comment:
why creating a dependency to `requireLatestFirmware` in your useMemo if it's always going to be `true` ? Is it for readability ?

Reply:
Yes mostly, and those changes are made to be a closer 100% match to [initSwap](https://github.com/LedgerHQ/ledger-live/blob/5eb574523c75c06c2f0209b33982b16149e6a375/libs/ledger-live-common/src/hw/actions/initSwap.ts#L151) where it is not a constant. 

I can remove if you think it is better